tests/integration/test_hnsw_psql.py

```diff
@@ -98,3 +99,51 @@ def test_basic_integration(docker_compose, get_documents):
             status
         )
         assert last_sync > last_sync_timestamp
+
+
+@pytest.mark.parametrize('docker_compose', [compose_yml], indirect=['docker_compose'])
+def test_replicas_integration(docker_compose, get_documents):
+    emb_size = 10
+    LIMIT = 10
+    NR_SHARDS = 2
+    docs = DocumentArray(get_documents(nr=100, emb_size=emb_size))
+
+    uses_with = {
+        'dim': emb_size,
+        'startup_sync': False,
+        'limit': LIMIT,
+    }
+
+    f = Flow().add(
+        name='indexer',
+        uses=HNSWPostgresIndexer,
+        uses_with=uses_with,
+        parallel=NR_SHARDS,
+        replicas=3,
+        # this will lead to warnings on PSQL for clashing ids
+        # but required in order for the query request is sent
+        # to all the shards
+        polling='all',
+        uses_after=MatchMerger
+    )
+
+    with f:
+        f.post('/index', docs)
+
+        search_docs = DocumentArray(
+            get_documents(index_start=len(docs), emb_size=emb_size))
```

Comment:
I am pretty sure, you haven't enabled black for this repository.

Reply:
done